src/server/req_runjob.c

```diff
@@ -265,6 +265,35 @@ clear_from_defr(int sd)
 
 /**
  * @brief
+ * 		wrapper function that calls process_hooks()
+ *
+ * 	@see
+ * 		req_runjob
```

Comment:
I believe these need to be on the same line.  You might as well put () at the end, so doxygen knows it is a function.

Reply:
I referred from other functions (i.e. process_hooks()) where this tag is being used in this format